packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -88,6 +88,7 @@ export default class GoogleCalendarService implements Calendar {
         reminders: {
           useDefault: true,
         },
+        guestsCanSeeOtherGuests: !calEventRaw.seatsHideAttendees,
```

Comment:
Google Cal is the only calendar service we integrate right now that can natively hide the guest list